tests/e2e/dpa_deployment_suite_test.go

```diff
@@ -66,7 +65,7 @@ func createTestDPASpec(testSpec TestDPASpec) *oadpv1alpha1.DataProtectionApplica
 			},
 		},
 		SnapshotLocations:    testSpec.SnapshotLocations,
-		UnsupportedOverrides: testSpec.UnsupportedOverrides,
+		UnsupportedOverrides: dpaCR.UnsupportedOverrides,
```

Comment:
You should be merging the two, or check that testSpec does not contain unsupportedOverrides since we are now no longer reading testSpec.UnsupportedOverrides.

Reply:
And by check I mean, a code in this func that errors out on testSpec.UnsupportedOverrides. but still provide some way of testing unsupportedOverrides in CI, not just manual Makefile.

Reply:
hrm.. I could merge, but which one wins if they both set the same image?  IMHO since we're going to use this feature in openshift/velero ASAP and mig-tools/oadp-non-admin ASAP it's ok to leave out.

Reply:
If we do ever get back to this, the testSpec wins since the cases with testSpec.unsupportedOverrides relies on those specific images. The rest will honor the read in environment for the rest of the test suite.. what we wanted for plugins/other repos tests.

Reply:
I can put it back and retest.  My old brain *thinks* it wasn't overwritten by the test spec.  I can look again

Reply:
the testSpec *should* win if we implement a merge functionality which isn't currently implemented.

Reply:
im not saying anything is overwritten by testspec.. just saying testspec should win if we ever do a merge as testSpec is higher priority/required by a single case whereas env is for everywhere else.

Reply:
then we need to ensure none of testSpec's are currently overriding what's in the env, I think. Either way it's not a problem imo now

Reply:
testSpec should override. testSpec is defined per case. We want ideally at least one case in oadp-operator main repo that check that override still works irregardless of what else is going on in other repos.

Reply:
There should not be more than one or two testSpec so the majority should be using env.